lib/bank_holidays.rb

```diff
@@ -5,13 +5,27 @@ module BankHolidays
   URL = 'https://www.gov.uk/bank-holidays/england-and-wales.json'
```

Comment:
We can remove the GOV.UK Parsing, if we use the gem for both GB and BE holidays.